flink-runtime-web/src/test/resources/rest_api_v1.snapshot

```diff
@@ -366,7 +366,8 @@
                     "type" : "string"
                   },
                   "optimizer_properties" : {
-                    "type" : "string"
+                    "type" : "object",
+                    "id" : "urn:jsonschema:org:apache:flink:runtime:rest:messages:JobPlanInfo:RawJson"
```

Comment:
I don't understand where does this `id` come from, could you explain?
It should be something like
`optimizer_properties : global_properties { ... }`

But with this change, it would be
`optimizer_properties : id { global_properties { ... } }`

Wouldn't it?

Reply:
I don't think so. `id` isn't a property of `optimizer_properties` but rather contains the schema path (since its an object now)
Objects have either `id` or a `$ref` to their schema (not sure when is each used)

Reply:
Thanks for clarifying!